source/spark_sql_migrations/tests/test_readme.py

```diff
@@ -0,0 +1,61 @@
+import re
+import os
+import tests.helpers.mock_helper as mock_helper
+from unittest.mock import Mock
+import spark_sql_migrations.container as container
+import spark_sql_migrations.schema_migration_pipeline as schema_migration_pipeline
+
+
+def _get_source_path() -> str:
```

Comment:
I'm a great fan of Uncle Bobs "step down rule" from his book "Clean Code".
[Here](https://medium.com/@seydialkan/clean-code-chapter-3-functions-158dad98bcb9) is a blog shortly describing it.